qa/rpc-tests/bip65-cltv-p2p.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python2
+#
```

Comment:
minor nit, just noticed that the copyright header was different in this file than the other